docs/designs/012-workflow-XID-13-and-31.md

```
@@ -0,0 +1,501 @@
+
+**Change:** Add GPC/TPC/SM to the metadata map in `createHealthEventFromResponse` method. The hardware location will be available from the parser response.
+
+```go
+func (xidHandler *XIDHandler) createHealthEventFromResponse(
+	xidResp *parser.Response, 
+	originalMessage string,
+) *pb.HealthEvents {
+	normPCI := xidHandler.normalizePCI(xidResp.Result.PCIE)
+	
+        // extract GPC, TPC and SM info from xidResp
+	if xidResp.Result.Metadata != nil {
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)